Gemfile

```diff
@@ -106,6 +106,8 @@ group :development, :test do
   gem "multi_json"
   gem "open_api-rswag-specs"
   gem "rswag"
+
+  gem "merge_db_schema"
 end
```

Comment:
Minor comment. Should it be in development only, rather than development and test?

Reply:
I actually have no idea, cough.

Reply:
But yes, moved to dev only as that makes more sense to me.